### PR TITLE
[ci skip] Updating the unix man page. Small typo.

### PR DESCRIPTION
**Pull request by wfpokorny**
Updating our unix man page to be current with recent 3.71 master.
Also small typo in processrenderoptions.cpp.

### CODE REVIEW

File: unix/povray.1 [hunk 1]
```diff
@@ -1,49 +1,41 @@
-.TH POV-Ray 1 "August 2013" "POV\-Team" "Version 3.7" \" -*- nroff -*-
+.TH POV-Ray 1 "December 2016" "POV\-Team" "Version 3.71" \" -*- nroff -*-
```
Comment (c-lipka) [inline unix/povray.1]: Should probably still be `3.7` - if the version number was intended to refer specifically to 3.7.0, as opposed to the "3.7 generation" as a whole, I would expect it to have read `3.7.0` all along.

**Comment by c-lipka** [comment]
How did you make the changes?

AFAIK the man pages used to be auto-generated from the HTML docs (though someone needs to dig up the script that does it, and run it).

File: unix/povray.1 (hunk 1 above)
Comment (wfpokorny) [inline unix/povray.1]: I was aiming for a man page which matched the 3.7.1 master branch if someone did a 'make install' after a build. I added -+CC added to the flags for example.  

I agree the previous should have been 3.7.0. 

Do you want something more generic? Only documentation correct for all 3.7.0 and 3.7.1 ?

**Comment by wfpokorny** [comment]
I did straight up edits and the testing with the usual 'man -l povray.1' 

It looked from the authorship in the file as if it was just edited, but if you find that script, let me know and  I'll close this pull req. For the record if a script, we should change it to reformat the authors section as it looks bad on the default 80 char width unix terminal & there was a formatting typo - or two - in the result.

**Pull request merged**